app.js

```diff
@@ -21,6 +23,10 @@ app.get('/', function(req, res, next){
   }))
 })
 
+app.get('/responses', function(req, res, next){
+  res.json(smartSurveyAPICall.getData(config.apiToken, config.apiTokenSecret, config.surveyID, smartSurveyAPICall.parseResponse))
```

Comment:
Do I need to just pass this to locals and then I can have access in the template?